mutt/idna.c

```diff
@@ -265,3 +272,25 @@ char *mutt_idna_local_to_intl(const char *user, const char *domain)
 
   return mailbox;
 }
+
+
+/*
+ * mutt_idna_print_version:    print idn or idn2 library version, which be called in version.c
+ *
+ * */
+const char *mutt_idna_print_version(void)
+{
+	static char vstring[STRING];
+
+  #ifdef HAVE_IDNA_H
+	snprintf(vstring, sizeof(vstring),
+			"libidn:%s", stringprep_check_version(NULL));
+  #elif defined(HAVE_IDN2_H)
+  	snprintf(vstring, sizeof(vstring),
+			"libidn2:%s", idn2_check_version(NULL));
+  #endif
```

Comment:
These don't print the same message as they used to.
The "compiled with" parts have gone.